source/scpd/Cpp.d

```diff
@@ -328,23 +328,44 @@ extern(C++, (StdNamespace)) extern(C++, class) struct vector (T, Alloc = allocat
 
         string toString() const @trusted
         {
-            bool first = true;
-            string ret = "[ ";
-            foreach (ref entry; this.constIterator())
+            static if (is(T : ubyte))
             {
-                if (!first)
-                    ret ~= ", ";
-                ret ~= entry.serializeToJsonString();
-                first = false;
+                import vibe.data.serialization : Base64ArrayPolicy;
+                import std.array : appender, Appender;
+
+                auto app = appender!string();
+                serializeWithPolicy!(JsonStringSerializer!(Appender!string), Base64ArrayPolicy)(this[], app);
+                return app.data;
+            }
+            else
+            {
+                bool first = true;
```

Comment:
Any idea why original code didn't just call the serialize method on this[] I think that is equivalent to the below verbose code. I left the old cold there for now... Please let me know if I should change it.